randovania/games/dread/presets/starter_preset.rdvpreset

```diff
@@ -3,7 +3,7 @@
     "base_preset_uuid": null,
     "name": "Starter Preset",
     "uuid": "407e97ed-decb-4889-8ab7-943f86ae48a4",
-    "description": "Basic preset.",
+    "description": "A beginner-friendly preset for Metroid Dread featuring full item randomization, moderate difficulty, and quality of life improvements like skippable cutscenes and optional hints.",
```

Comment:
Not quite a fan of the phrasing here
- moderate difficulty makes it sound like tricks are enabled. The starter preset is the most basic/easiest preset rdv can provide here. So you literally cannot make it easier for the player
- I wouldn't call the hints optional, to encourage players to use them.

Reply:
More thoughts:
- full item randomization is kind of a given. It would be weird if it *wasnt* providing that.
- iirc dread starter preset starts with scan pulse? (Or however its named, can never remember the difference in name between samus returns and dread...)

Reply:
> More thoughts:
> 
>     * full item randomization is kind of a given. It would be weird if it _wasnt_ providing that.
> 
>     * iirc dread starter preset starts with scan pulse? (Or however its named, can never remember the difference in name between samus returns and dread...)

It's "Pulse Radar" in Dread.

Reply:
I also wouldn't mention skippable cutscenes since those aren't even an option, they're always on.

This description should be saying what the preset does, not what the rando does.

And yes, this preset starts with Pulse Radar, which should be mentioned.

Reply:
Also, for the record, hints aren't Quality of Life. They're a core part of the randomizer's design. You also can't turn those off in Dread, like skippable cutscenes.

Reply:
Also should definitely mention the 3 DNA goal.

Reply:
You can look at the starter presets for samus returns / prime 1 / echoes / am2r for a bit of inspiration.

Reply:
> Not quite a fan of the phrasing here
> - moderate difficulty makes it sound like tricks are enabled. The starter preset is the most basic/easiest preset rdv can provide here. So you literally cannot make it easier for the player
> - I wouldn't call the hints optional, to encourage players to use them.

I think full item rando might make sense from a non-metroid rando player. Like how iirc zootr has separate settings to use keysanity, or how hollow knight has added... Pretty much everything to the item pool by this point lmao. I agree the phrasing is a bit weird but it would probably help clarify for people with experience in other game randos.

Reply:
In Randovania we have "player has 100% the game at least once" as an expectation.
By that standard, the player should know imo that theres nothing in dread worth putting into seperate pools. 
Even if you haven't 100%ed it and just did a normal finish, the game should've made it clear already that it doesn't have bigger sidequests or very big secrets for collectibles.

Reply:
> > Not quite a fan of the phrasing here
> > 
> > * moderate difficulty makes it sound like tricks are enabled. The starter preset is the most basic/easiest preset rdv can provide here. So you literally cannot make it easier for the player
> > * I wouldn't call the hints optional, to encourage players to use them.
> 
> I think full item rando might make sense from a non-metroid rando player. Like how iirc zootr has separate settings to use keysanity, or how hollow knight has added... Pretty much everything to the item pool by this point lmao. I agree the phrasing is a bit weird but it would probably help clarify for people with experience in other game randos.

that's not what the preset description is for, though. that information is conveyed elsewhere; the preset description should be describing the settings chosen for a given preset and its purpose for being an included preset

Reply:
"A beginner-friendly preset for Metroid Dread. This preset provides a straightforward experience by starting with the Pulse Radar and having all items randomized. It is designed to be the easiest preset available, ideal for players new to randomizers. The goal is set to 3 DNA, and hints are integrated as a core part of gameplay to guide progression" how about using this.

Reply:
> ideal for players new to randomizers

This part feels very off to me, considering dread is not the best game to jump in if you're new to randomizers. I'd phrase it more as "new to this/Dread's randomizer"

Unsure about the rest, will leave that to more dread-experienced people.

Reply:
> This preset provides a straightforward experience by starting with the Pulse Radar and having all items randomized. It is designed to be the easiest preset available, ideal for players new to randomizers. 

This preset aims to help players who are less experienced with Metroid Dread by providing them with the Pulse Radar at the start of the game.

> The goal is set to 3 DNA, and hints are integrated as a core part of gameplay to guide progression

In order to access Raven Beak, you must collect all Metroid DNA. This preset requires collecting 3 DNA which is placed on E.M.M.I. and bosses. Hints to where the DNA can be found is located in Itorash and Dairon.

Reply:
Ah yeah that's no good then

Reply:
New suggestion:

"This preset is designed for those new to Metroid Dread or its randomizer. Every seed can be completed without utilizing any tricks or glitches.

Along with shuffling items, some changes have been made to improve the randomizer experience. These include starting with Pulse Radar, preventing some softlocks, and more. This preset requires collecting 3 Metroid DNA in order to access Raven Beak."

Reply:
Is there an update on this? @Swaraj2259